src/web/components/SubNav/SubNav.tsx

```diff
@@ -163,7 +163,7 @@ export const SubNav = ({ subNavSections, format, currentNavLink }: Props) => {
 		} else {
 			setShowMore(false);
 		}
-	}, [ulRef, setShowMore]);
+	}, [ulRef]);
```

Comment:
Good call. I _think_ we can also remove `ulRef` from the deps here too? I don't have a complete mental model for what should go here though but I notice the lint rule is okay with it not being present

Reply:
ah yes - I think these are guaranteed by React to not change so both can be removed.